README.md

```diff
@@ -14,11 +15,11 @@ and Pereira, J. M. B.<sup>3</sup>
 de Janeiro, RJ, Brazil.*
 
 This paper has been published in 
-[**Geochemistry, Geophysics, Geosystems (G<sup>3</sup>)**](https://doi.org/10.1002/2016GC006329).
+[**Geochemistry, Geophysics, Geosystems (G<sup>3</sup>)**](http://agupubs.onlinelibrary.wiley.com/hub/journal/10.1002/(ISSN)1525-2027).
 
-doi: [10.1002/2016GC006329](https://doi.org/10.1002/2016GC006329)
+doi: [10.1002/2016GC006329](http://onlinelibrary.wiley.com/doi/10.1002/2016GC006329/abstract)
 
-A PDF version is available at [pinga-lab.org/papers/paper-magnetization-rock-sample-2016.html](http://www.pinga-lab.org/papers/paper-magnetization-rock-sample-2016.html)
+A PDF version is available at [pinga-lab.org/pdf/paper-magnetization-rock-sample-2016.pdf](http://www.pinga-lab.org/pdf/paper-magnetization-rock-sample-2016.pdf)
```

Comment:
@andrelreis Don't link directly to PDFs. They aren't part of the group page and so people won't be able to see other works by the group. That's the point of the website. When someone visits they can click on your name and see what else you published.